projects/dynamicforms/src/components/upload/upload.component.html

```diff
@@ -43,7 +43,9 @@
                     ><i class="fas fa-check"></i
                   ></span>
                   <span class="col-8 text-break">{{ file?.name }}</span>
-                  <span class="col-3"> {{ convertFileSize(file.size) }}</span>
+                  <span class="col-3">
+                    {{ file.size ? convertFileSize(file.size) : '' }}</span
```

Comment:
Maybe we could leave it the way it was just with `convertFileSize(file?.size)` and in the `convertFileSize` method itself set the default argument, just in case size is undefined. Like so:
`convertFileSize(bytes: number = 0)`. This way the logic would stay completely in the method.